ingestion/src/metadata/ingestion/stage/table_usage.py

```diff
@@ -132,6 +141,10 @@ def get_status(self):
 
     def close(self):
         for key, value in self.table_usage.items():
+            try:
+                value.sql_queries = self.table_queries[key]
+            except KeyError:
+                pass
```

Comment:
What is the purpose of this? Looks like it is doing nothing 🤔

Reply:
@ulixius9 we can use .get here, as it will return None as fallback

Reply:
Oh, then it must be me missing something. Does the stage return a value during the `close`? Or how does it pass things down the workflow?

Reply:
oooh writes when closing my bad. This is OK